services/movie_session.py

```diff
@@ -0,0 +1,51 @@
+import init_django_orm  # noqa: F401
+
+from datetime import datetime
+from django.db.models import QuerySet
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+
+
+def get_movies_sessions(
+        session_date: datetime = None
+) -> QuerySet:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(
+        movie_session_id: int
+) -> QuerySet:
```

Comment:
when you get one movie session - return type of annotation should be class instance, not queryset